src/App.tsx

```diff
@@ -21,75 +21,139 @@ enum SortType {
   LENGTH,
 }
 
-type ReorderOptions = {
-  sortType: SortType,
-  isReversed: boolean,
-};
+// type ReorderOptions = {
+//   sortType: SortType,
+//   isReversed: boolean,
+// };
 
 // Use this function in the render to prepare goods
-export function getReorderedGoods(
-  goods: string[],
-  { sortType, isReversed }: ReorderOptions,
-) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+// export function getReorderedGoods(
+//   goods: string[],
```

Comment:
Please delete all the comments that you don't need

Reply:
fix it, please